docs/walkthrough/finetuning-models.md

```diff
@@ -1,10 +1,128 @@
 ---
-sidebar_position: 2
+sidebar_position: 4
 ---
 
 # Finetuning Models
 
 <!-- THE MARKDOWN (.md) FILE IS GENERATED FROM THE NOTEBOOK (.ipynb) FILE -->
+The large pretrained base models are already very helpful, but
+with finetuning smal and large language models can be made even more helpful.
```

Comment:
I think we might need to include a spellcheck linter in our generation step to catch typos like `smal and large`.

Reply:
it's just a smol typo tho.

I use [this extension](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) and confine it to only some filetypes with the following in my vscode settings.json:
```
"cSpell.enabledLanguageIds": [
    "git-commit",
    "json",
    "jupyter",
    "markdown",
    "plaintext",
    "restructuredtext",
    "scminput",
    "text",
    "yaml",
    "yml"
  ]
```

Reply:
I installed a spellchecker and fixed the typos. I do think having a linter built-in as a command would be cool too but outside scope of this PR